javascript.md

```diff
@@ -659,7 +663,28 @@ console.log(index); // outputs 1
 Rust:
 
 ```rust
-// TODO
+#[derive(Debug, Clone, Copy)]
+struct Foo { pub foo: i32 }
+
+let list = vec![
+    Foo { foo: 1 },
+    Foo { foo: 3 },
+    Foo { foo: 2 },
+];
+let obj3 = list.iter().find(|item| item.foo == 2);
+println!("{:?}", obj3); // outputs Some(Foo { foo: 2 })
+let index = list.iter().position(|item| item.foo == 3);
+println!("{:?}", index); // outputs Some(1)
+```
+
+Rust, but using tuples to make ad-hoc objects instead of having to define a struct:
```

Comment:
Hm, I'm on the fence about this variant with tuples-- I don't think I'd write code like this, and it doesn't seem exactly equivalent to the javascript because "foo" isn't really a key/property name/field. Do you think this would be important to javascript programmers to ease their transition into Rust?

Reply:
I completely agree. That was just weird; not sure what I was thinking at the time. :D Fixed in 0f9d43b.